src/components/ModelSteps/Contact.jsx

```diff
@@ -22,72 +24,111 @@ import CompositeHeader from 'src/components/CompositeHeader/CompositeHeader'
 const Contact = () => {
   const client = useClient()
   const { t } = useI18n()
-  const [currentUser, setCurrentUser] = useState(null)
+  const { setFormData } = useFormData()
+  const [contactModalOpened, setContactModalOpened] = useState(false)
+  const [contactsList, setContactsList] = useState([])
+  const [contactIdSelected, setContactIdSelected] = useState(null)
 
   useEffect(() => {
     let isMounted = true
     ;(async () => {
-      const user = await fetchCurrentUser(client)
-      isMounted && setCurrentUser(user)
+      const myself = await fetchCurrentUser(client)
+      if (isMounted) {
+        setContactsList([myself])
+        setContactIdSelected(myself._id)
+      }
     })()
 
     return () => {
       isMounted = false
     }
   }, [client])
 
+  useEffect(() => {
+    if (contactIdSelected) {
+      setFormData(prev => ({
+        ...prev,
+        contacts: contactsList.filter(
+          contact => contact._id === contactIdSelected
+        )
+      }))
+    }
+  }, [contactIdSelected, contactsList, setFormData])
+
+  const onChangeRadio = evt => setContactIdSelected(evt.target.value)
+
+  const onClickContactsListModal = contact => {
+    const contactAlreadyListed = contactsList.some(cl => cl._id === contact._id)
+    if (!contactAlreadyListed) {
+      setContactsList(prev => [...prev, contact])
+    }
+    setContactIdSelected(contact._id)
+    setContactModalOpened(false)
+  }
+
   return (
-    <Paper elevation={2} className={'u-mt-1'}>
-      <List>
-        <ListItem>
-          <ListItemIcon>
-            <Avatar
-              size={'small'}
-              style={{
-                color: 'var(--primaryColor)',
-                backgroundColor: 'var(--primaryColorLightest)'
-              }}
-            />
-          </ListItemIcon>
-          <ListItemText
-            primary={`${t('ContactStep.me')} ${
-              currentUser?.fullname ? `(${currentUser.fullname})` : ''
-            }`}
-          />
-          <ListItemSecondaryAction className={'u-mr-half'}>
-            <Radio defaultChecked />
-          </ListItemSecondaryAction>
-        </ListItem>
+    <>
+      <Paper elevation={2} className={'u-mt-1'}>
+        <List>
+          <div className={'u-mah-5 u-ov-scroll'}>
+            {contactsList.map(contact => (
+              <ListItem
+                key={contact._id}
+                onClick={() => setContactIdSelected(contact._id)}
+              >
+                <ListItemIcon>
+                  <Avatar
+                    size={'small'}
+                    style={{
+                      color: 'var(--primaryColor)',
+                      backgroundColor: 'var(--primaryColorLightest)'
+                    }}
```

Comment:
You should not create an inline style in a render method. Already explained. With that, at each re-render of the Contact component, we create a new style object meaning that we re-render also the Avatar component

Reply:
Indeed, it's being changed, this code is more than 2 months old 👍